modules/network/private-endpoint/main.bicep

```diff
@@ -0,0 +1,78 @@
+@description('Required. The name of the Private Endpoint.')
+param name string
+
+@description('Required. Location for the Private Endpoint.')
+param location string
```

Comment:
it can use a default as `resourceGroup().location`, as per https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md#parameters